### PR TITLE
Fix burger menu display on mobile search

### PR DESCRIPTION
**Pull request by maudetes**
Fix https://github.com/etalab/data.gouv.fr/issues/349.

Maybe not the most graceful fix?

### CODE REVIEW

File: udata_gouvfr/theme/gouvfr/templates/header.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
                         @click.capture.prevent="$refs.suggest.startSearch()">
                         {% include theme('svg/search.svg') %}
                     </a>
-                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open')"
+                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open'); if ($refs.suggest.active) {$refs.suggest.stop();}"
```
Comment (abulte) [inline udata_gouvfr/theme/gouvfr/templates/header.html:35]:
```suggestion
                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open'); if ($refs.suggest.active) $refs.suggest.stop();"
```

Comment (abulte) [reply udata_gouvfr/theme/gouvfr/templates/header.html:35]: Maybe add a comment to explain what this does, because I have no idea right now :-)

Comment (maudetes) [reply udata_gouvfr/theme/gouvfr/templates/header.html]: Yep, I can add a comment, but it is supposed to deactivate the suggest view when opening the burger menu.

**Comment by maudetes** [comment]
However, the suggest view seems quite buggy on mobile (and currently not functional https://github.com/etalab/data.gouv.fr/issues/359), maybe we should deactivate it entirely on mobile for now (making this PR not relevant anymore) ?

**Comment by abulte** [comment]
Seems like a quick win for now (removing suggest on mobile) 👌, until @gmougeolle gets its things straight 🙃

**Comment by maudetes** [comment]
Closing this PR as we're going to deactivate suggest view for now on mobile. See https://github.com/etalab/data.gouv.fr/issues/360.

**Pull request closed without merging**